tsc/meetings/2019-02-14.md

```diff
@@ -0,0 +1,120 @@
+
+5) Dan: three points in favor of Travis/Circle/Github CI. The first is that the
+current Jenkins setup uses an additional repository for the CI build system,
+where as Travis uses the CI included in the OpenVDB repository. This is a
+limitation of the way that Jenkins is currently configured in the ASWF. Having
+the CI in the same repo makes it easier to sync, jump around the history, merge,
+fork etc. Thanh mentioned that there is a way that Jenkins might be setup to
+accommodate this, but it uses Jenkins's own proprietary systems. Second, the
+logs in Travis have a much nicer layout to Jenkins. Third, Travis has inbuilt
+ccache support. We also already have Travis integration. The additional
+computational resources are the main benefit of the current Jenkins system.
+Thanh has been investigating Circle CI and another LF group (the LF Networking
+group) has been looking at alternatives. This group meets Fridays.
+
+Meeting notes from the Tooling working group:
+
+https://wiki.lfnetworking.org/display/LN/LFN+Infra+Work+Group+1+Feb+19
+
+Meeting schedule/mailing list for the working group for anyone who's interested
+in attending:
+
+https://lists.lfnetworking.org/g/TAC/topic/lfn_tooling_subgroup/29576852?p=
+
+Note that CMake support is unrelated to the CI and should be prioritized.
+
+6) Main point of entry for reporting bugs isn't clear. There are four places:
+JIRA, Github issues, e-mail, forum. The forum has recently been getting spam.
+Membership is required for the forum which Ken has to confirm. We could enable
+moderation so that posts have to be approved before they are visible. Jeff
+mentioned that SideFX only moderate first posts.
+
+Dan: Github issues are nice as they allow you to close the issue/mark as resolved.
+Nick: the lack of documentation exacerbates the problem of potentially trivial
+forum posts and dilutes the forum. Nick brought up the current issue with external
+(non-TSC) pull requests stagnating. It was agreed that we need a Github check for
+CLA. Dan mentioned that it might be worth consolidating the current contributing
+.md files to have a clearer documented process for external PRs.
+
+Dan's message on the above: https://lists.aswf.io/g/openvdb-dev/message/41
+
+7) Jeff used the current statistics header in OpenVDB to introduce VDB reduction
+support (for volumes, not points) in Houdini 17.5. Nick asked if the hscript
+expression might also benefit from supporting VDBs.
+
+Peter mentioned that the process functions in Houdini which operate on set types
+of VDB grids could be changed to take a tuple of types to allow for cleaner
+instantiation of supported grid types across Houdini SOPs.
+
+TAC meeting discussed versioning. All projects should be using the same type of
+versioning. Semantic versioning was the common thread.
+
+OpenVDB ABI only for grid compatibility. Nick asked if this was written down
+anywhere, it doesn't seem to be. Dan mentioned the issue with unsupported file
```

Comment:
I believe this is documented here:

https://www.openvdb.org/documentation/doxygen/faq.html#sVersionNumbering

And in the Doxygen documentation in version.h:

https://www.openvdb.org/documentation/doxygen/version_8h.html

I agree that it could be more prominent though.

Reply:
our semantic versioning scheme is also discussed in openvdb/version.h but I agree that we should put it in a centralized place (maybe even the web-site)